tests/Components/DeleteUserFormTest.php

```diff
@@ -20,8 +20,26 @@
         ->test(DeleteUserForm::class)
         ->assertViewIs('ark-fortify::profile.delete-user-form')
         ->call('confirmUserDeletion')
-        ->assertSee('Are you sure you want to delete your account? Deleting your account is irreversible and all deleted data is unrecoverable')
+        ->assertSee('After 30 days your account will be permanently deleted and become unrecoverable. To confirm this action, enter your username below.')
```

Comment:
use the lang file instead of the text so you don't have to update this if the text changes